lib/bridge-client/index.js

```diff
@@ -40,6 +42,12 @@ var ExchangeReport = require('./exchange-report');
  * @param {Object} options.basicAuth
  * @param {String} options.basicAuth.email - Email address for HTTP basic auth
  * @param {String} options.basicAuth.password - Password for HTTP basic auth
+ * @param {String} [options.blacklistFolder] - The folder that blacklist
+ * entries will be persisted to if using the default fs-store
+ * @param {Object} [options.store] - The store that blacklist enteries will be
+ * persisted to if using the default fs-store
+ * persisted to. This object must be compatible with the API of
+ * [abstract-blob-store](https://github.com/maxogden/abstract-blob-store)
```

Comment:
For links in JSDoc comments, use `{@link https://github.com/maxogden/abstract-blob-store|abstract-blob-store}` to render it properly on the docs sits.